app/CL_EditorImpl/src/main/java/org/jjazz/cl_editorimpl/BR_Chords.java

```diff
@@ -158,13 +161,25 @@ public Position getPositionFromPoint(int x)
     }
 
     @Override
-    public void showInsertionPoint(boolean b, ChordLeadSheetItem<?> item, Position pos, boolean copyMode)
+    public int getDisplayTransposition()
+    {
+        return displayTransposition;
+    }
+
+    @Override
+    public void setDisplayTransposition(int dt)
+    {
+        displayTransposition = dt;
```

Comment:
The update of ItemRenderers must be done here in BR_Chords, not in BarBox anymore. Otherwise BR_Chords API will NOT work if it is used without a BarBox (eg if you add an item it won't be transposed despite having called setDisplayTransposition()).